frontend/src/js/tooltip/AdditionalInfoHoverable.tsx

```diff
@@ -15,36 +16,39 @@ const Root = styled("div")`
 // Allowlist the data we pass (especially: don't pass all children)
 const getAdditionalInfos = (
   node: ConceptT,
-  parent?: ConceptT,
+  root?: ConceptT,
 ): AdditionalInfosType => ({
   label: node.label,
   description: node.description,
-  isFolder: !!node.children && node.children.length > 0,
   matchingEntries: node.matchingEntries,
   matchingEntities: node.matchingEntities,
   dateRange: node.dateRange,
   infos: node.additionalInfos,
-  parent: parent,
+  icon: getNodeIcon(node, {
+    isStructNode: !root?.detailsAvailable,
+  }),
+  rootLabel: root?.label,
+  rootIcon: root ? getNodeIcon(root) : undefined,
 });
 
 const AdditionalInfoHoverable = ({
   node,
   className,
   children,
-  parent,
+  root,
 }: {
   children: ReactNode;
-  node: ConceptT;
   className?: string;
-  parent?: ConceptT;
+  node: ConceptT;
+  root: ConceptT;
 }) => {
   const dispatch = useDispatch();
 
   const onDisplayAdditionalInfos = () => {
     if (!node.additionalInfos && isEmpty(node.matchingEntries)) return;
     dispatch(
       displayAdditionalInfos({
-        additionalInfos: getAdditionalInfos(node, parent),
+        additionalInfos: getAdditionalInfos(node, root),
```

Comment:
calling it `root`, not `parent`, because strictly saying, it's not really the parent